extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -166,6 +166,14 @@ export class SubscriptionService {
         });
 
       isStoryDocument(this.ampdoc_).then((isStory) => {
+        if (isStory) {
+          // Make the dialog with round corners for AMP Story.
+          const doc = this.ampdoc_.getRootNode();
+          const dialogWrapperEl = doc.querySelector('amp-subscriptions-dialog');
+          dialogWrapperEl.classList.add(
+            'i-amphtml-story-subscriptions-dialog-wrapper'
+          );
+        }
```

Comment:
You're not using this class, does it really belong in this PR?
Also, what happens if `amp-subscriptions-dialog` is NOT specified?